contracts/MintBurnERC20.sol

```diff
@@ -0,0 +1,25 @@
+//SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+
+contract MintBurnERC20 is ERC20, AccessControl {
+    bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");
+    bytes32 public constant BURNER_ROLE = keccak256("BURNER_ROLE");
```

Comment:
I think one role is enough for both mint and burn - it's the same level of access, controlling supply.
Also - burning, kinda every token has burn defined for the account owner

Reply:
Burner_ROLE removed and used ERC20Burnable instead.